src/test/TestArgs.hpp

```diff
@@ -47,6 +47,7 @@ class TestArgs
     llvm::cl::opt<bool>         unitOption;
     llvm::cl::list<std::string> inputPaths;
     llvm::cl::opt<std::string>  addons;
+    llvm::cl::list<std::string> libCxxPaths;
```

Comment:
There's a 1-to-1 relationship between Config and ToolArgs. Conceptually, they both derive from ConfigOptions.inc. We have yet to be able to automate all steps, but the logic relies on that premise. The thing can't be in Config and TestArgs because they mean different things.

Reply:
I noticed that `addons` is included in both `Config` and `TestArgs`, similar to my addition of `stdLibPaths` (aka: `libCxxPaths`). Given your feedback on avoiding duplication between these, should we revisit the placement of `addons`?